lib/game.rb

```diff
@@ -0,0 +1,117 @@
+
+
+class Game
+
+  attr_accessor :board, :player_1, :player_2
+
+WIN_COMBINATIONS = [
+  [0,1,2], 
+  [3,4,5], 
+  [6,7,8], 
+  [0,3,6],
+  [1,4,7],
+  [2,5,8],
+  [0,4,8],
+  [6,4,2]
+]
+
+
+def initialize(player_1=Human.new("X"), player_2=Human.new("O"), board=Board.new)
+  @board = board
+  @player_1 = player_1
+  @player_2 = player_2
+end
+
+def current_player
+  board.turn_count.even? ? player_1 : player_2
+end
+
+def over?
+  draw? || won?
+end
+
+def won?
+    WIN_COMBINATIONS.detect do |combo|
+      board.cells[combo[0]] == board.cells[combo[1]] &&
+      board.cells[combo[1]] == board.cells[combo[2]] &&
+      board.cells[combo[0]] != " "
+    end
+  end
+
+def draw?
+  board.full? && !won?
+end
+
+def winner
+  board.cells[won?[0]] if won?
```

Comment:
Instead of calling `Game#won?` twice, consider using an if statement assignment. For example:

```ruby
if won = won?
  board.cells[won.first]
end
```

If `won?` returns true, then and only then will the local variable `won` be assigned, otherwise, the method will just abort with a false return value. But if `won` is assigned, we can use that variable instead of calling `won?` again within the if statement.